server/aspnetcore/TicketHub/Controllers/TicketController.cs

```diff
@@ -0,0 +1,110 @@
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using System.Net.Mime;
+using System.Security.Claims;
+using TicketHub.Database;
+
+namespace TicketHub.Controllers;
+
+[ApiController]
+[Route("/api")]
+[Produces("application/json")]
+[Consumes(MediaTypeNames.Application.Json)]
+public class TicketController : ControllerBase
+{
+    private readonly ILogger<TicketController> _logger;
+    private readonly PostgresContext _dbContext;
+
+    public record TicketFields(string customer, string description);
+    public record ResolveFields(bool resolved);
+
+    private async Task<Customer> addCustomer(Tenant tenant, string name)
+    {
+        Customer c = new Customer();
+        c.Name = name;
+        c.Tenant = tenant.Id;

```

Comment:
Consider adding a `Customer` constructor so that this may be rewritten:
```suggestion
        Customer c = new Customer(tenant.Id, name);
```

The appropriate constructor in Customer.cs would be:
```
    public Customer(int tenant, string? name)
    {
        Tenant = tenant;
        Name = name;
    }
```

Reply:
Ah! Good catch. I'll integrate that shortly. :smile: